### PR TITLE
feat: added i18n packages

### PR DESCRIPTION
**Pull request by kyranet**
Featuring `@skyra/http-framework-i18n` and `@skyra/i18next-backend`.

### CODE REVIEW

File: packages/http-framework-i18n/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# `@skyra/http-framework-i18n`
+
+An internationalization layer powered by [`i18next`](https://www.npmjs.com/package/i18next) for [`@skyra/http-framework`](https://www.npmjs.com/package/@skyra/http-framework).
```
Comment (favna) [inline packages/http-framework-i18n/README.md:3]: Needs more info

File: packages/http-framework-i18n/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		"watch": "tsc -b src --watch",
+		"watch:types": "tsc -b src -w",
+		"lint": "eslint src --ext ts --fix -c ../../package.json",
+		"prepublishOnly": "yarn build"
+	},
+	"dependencies": {
+		"@discordjs/collection": "^0.5.0",
+		"@sapphire/utilities": "3.6.2",
+		"@skyra/http-framework": "workspace:^",
+		"@skyra/i18next-backend": "workspace:^",
```
Comment (favna) [inline packages/http-framework-i18n/package.json:26]:
```suggestion
		"@skyra/http-framework": "*",
		"@skyra/i18next-backend": "*",
```

File: packages/i18next-backend/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# `@skyra/i18next-backend`
+
+An internationalization layer powered by [`i18next`](https://www.npmjs.com/package/i18next) and [`@skyra/i18next-backend`](https://www.npmjs.com/package/@skyra/i18next-backend) for [`@skyra/http-framework`](https://www.npmjs.com/package/@skyra/http-framework).
```
Comment (favna) [inline packages/i18next-backend/README.md:3]: Needs more info

**Review by favna — CHANGES_REQUESTED** [review]

File: packages/http-framework-i18n/package.json (hunk 2 above)
Comment (kyranet) [reply packages/http-framework-i18n/package.json:26]: Why? This works flawless with Turbo.

Comment (favna) [reply packages/http-framework-i18n/package.json:26]: Because it won't work flawlessly when published to npm.....

Comment (kyranet) [reply packages/http-framework-i18n/package.json:26]: Then how does Discord.js do it? They use `workspace:^` and the published versions have the correct versions to the other packages.

Comment (favna) [reply packages/http-framework-i18n/package.json:26]: I have no idea but `workspace:^` is gone in the published bundle: https://unpkg.com/browse/discord.js@14.0.0-dev.1649505803-3c0bbac/package.json

Comment (kyranet) [reply packages/http-framework-i18n/package.json:26]: Exactly, and the actual code uses it: https://github.com/discordjs/discord.js/blob/main/packages/discord.js/package.json#L50-L52

**Review by favna — APPROVED** [review]

**Pull request merged**